src/components/Comparemodal.js

```diff
@@ -0,0 +1,210 @@
+                  onClick={handleClose}
+                >
+                  <span aria-hidden='true'>&times;</span>
+                </button>
+              </div>
+
+              <div className='modal-body'>
+                <div
+                  className={
+                    modal['images-container'] + ' ' + modal['general-container']
+                  }
+                >
+                  <div className={general['image-container']}>
+                    <img
+                      className={general['pokemon-image']}
+                      alt={select.currentPokemon}
+                      src={
+                        select.descriptions[select.currentIndex]
+                          .sprites.front_default
+                      }
+                    />
+                  </div>
+                  <div className={general['image-container']}>
+                    <img
+                      className={general['pokemon-image']}
+                      alt={compare.compareTo.name}
+                      src={compare.compareTo.sprites.front_default}
+                    />
+                  </div>
+                </div>
+                <div className={modal['compare-container']}>
+                  <div className={modal['compare-description']}>
+                    <div className={modal['compare-text']}>
+                      {
+                        select.descriptions[select.currentIndex]
+                          .height
+                      }
+                      m
+                    </div>
+                    <div className={modal['compare-text']}>
+                      {
+                        select.descriptions[select.currentIndex]
+                          .weight
+                      }
+                      kg
+                    </div>
+                    <div className={modal['compare-text']}>
+                      {select.genders[select.currentIndex]}
+                    </div>
+
+                    {select.descriptions[
+                      select.currentIndex
+                    ].abilities.map((ability) => (
+                      <div
+                        className={modal['compare-text']}
+                        key={ability.ability.url}
+                      >
+                        {ability.ability.name}
+                      </div>
+                    ))}
+                  </div>
+                  <div className={modal['compare-description']}>
+                    <div className={modal['modal-subtitle']}>Height</div>
+                    <div className={modal['modal-subtitle']}>Weight</div>
+                    <div className={modal['modal-subtitle']}>Gender</div>
+                    <div className={modal['modal-subtitle']}>Abilities</div>
+                  </div>
+                  <div className={modal['compare-description']}>
+                    <div className={modal['compare-text']}>
+                      {compare.compareTo.height}m
+                    </div>
```

Comment:
here you still have the same issue, there are three columns with information, if you want to align them and there is one that has 2 line for some reason you will not be able to do it